aws-ts-twitter-athena/index.ts

```diff
@@ -0,0 +1,123 @@
+import * as pulumi from "@pulumi/pulumi";
```

Comment:
Should we just delete `cloud-js-twitter-athena` now?  It was explicitly AWS-specific, and didn't take advantage of anything that is unique to `cloud` - so I see no reason we wouldn't prefer folks to look at this example instead of that one.